### ISSUE TITLE
Running `Fix Imports` leaves files unsaved

### ISSUE THREAD

**Issue by emme1444** [luanpotter/vscode-dart-import#32]
When invoking the `Fix Imports` command files are processed, but then left unsaved (still showing a dot). Only to have to save them again. I'm mostly using this with the `fixOnSave` option **_enabled_**. That's why it's bothering me. Meaning I have to save all files twice every time. Not too bothersom, but it adds up and is not that clean in my opinion. Here's an example: 

I don't know if this is because it's running the `Organize Imports` action. Couldn't tell since I haven't looked into the code, but might be something to do with that. I don't know, but feels like the `vscode` api would expose some method of saving the file. Maybe that's sufficient? 

Anyway, hope we can fix this!  
Let me know if  there's anything else.

**Comment by luanpotter** [comment]
Agreed it should save, if anyway wants to help with a fix that would be great! :)

**Comment by allanwolski** [comment]
It would be great!
